SMLHelper/Patchers/ItemActionPatcher.cs

```diff
@@ -0,0 +1,151 @@
+﻿using Harmony;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text;
+
+namespace SMLHelper.V2.Patchers
+{
+    internal class ItemActionPatcher
+    {
+        internal class CustomItemAction
+        {
+            internal Action<InventoryItem> Action;
+            internal string LanguageLine;
+            internal Predicate<InventoryItem> Condition;
+
+            internal CustomItemAction(Action<InventoryItem> action, string tooltip, Predicate<InventoryItem> condition)
+            {
+                Action = action;
+                LanguageLine = tooltip;
```

Comment:
For clarity, we might want to rename this internal field to be `LanguageLineKey` and the incoming parameter to be `tooltipKey`, since this is how they are being used with the Language lines.